### PR TITLE
Add endpoints for updating tags

### PR DESCRIPTION
**Pull request by tenyo**
- Also update license to AGPL-3.0

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -117,6 +117,9 @@ func main() {
 	// Service Orchestration handlers
 	api.HandleFunc("/{account}/services", ServiceOrchestrationCreateHandler).Methods(http.MethodPost)
 	api.HandleFunc("/{account}/services", ServiceOrchestrationDeleteHandler).Methods(http.MethodDelete)
+	api.HandleFunc("/{account}/services", func(w http.ResponseWriter, req *http.Request) {
+		w.WriteHeader(http.StatusNotImplemented)
```
Comment (fishnix) [inline main.go]: you might still want to do the stuff to check if the account exists?

**Comment by fishnix** [comment]
except that one thing, looks good 👍

File: main.go (hunk 1 above)
Comment (tenyo) [reply main.go]: ok, i created a handler for that endpoint

**Comment by fishnix** [comment]
👍

**Comment by damnski** [comment]
looks good

**Pull request merged**